docs/specs/tripgo.swagger.yaml

```diff
@@ -3411,6 +3411,12 @@ definitions:
           - STOP_BUTTON
           - TURN_ON_LIGHT
           - MOBILE_APP
+      pickUpOnly:
+        type: boolean
+        description: Is this stop pick up only? Missing when unknown or false.
```

Comment:
I don't like that missing can mean unknown or false. Why not always return it, if we know and then have it say `true` or `false`.

Reply:
You are right. I have just fixed it.